src/utils/dateHelpers.ts

```diff
@@ -2,3 +2,12 @@ import dayjs from 'dayjs';
 
 export const getNextMonday = () =>
   dayjs().utc().startOf('week').add(1, 'week').day(1).format('MMMM D');
+
+export const timestampToDateTimeString = (timestamp: number) =>
```

Comment:
You still use another method (`timestampToString`) in `useGetSaleInformation` for this. Use this method instead.

Reply:
**timestampToString** and **timestampToDateTimeString**  return the different result.
**timestampToString** - returns month and date only
**timestampToDateTimeString** - returns monty, date, hours and minutes.
I moved **timestampToString** to the **dateHelpers** and imported it in **useGetSaleInformation**.